docs/reference/types.md

```diff
@@ -966,6 +966,21 @@ Return a new array with the same items, but sorted.
   If given, applies this function to the elements in the array to determine the keys to sort by.
 - returns: array
 
+### dedup()
+Returns a new array with all duplicate items removed.
+
+Only the first element of each duplicate is kept.
+
+```example
+#{
+  (1, 1, 2, 3).dedup() == (1, 2, 3)
```

Comment:
how about adjusting the example such that we can see that even non-consecutive duplicates are removed? like adding a 1 at the end. and maybe make it not sorted to show that it works generally.